.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: go
```

Comment:
https://raw.githubusercontent.com/030/go-yq/master/.travis.yml

Reply:
Update the content to match your other travis file